### PR TITLE
Se 3172

### PR DESCRIPTION
**Pull request by kwcdbc**
First go at javascript handling of work data
Not yet ready for merge yet. 
Only look at javacript code commits by me.

### CODE REVIEW

File: javascript/src/main/resources/javascript/Demo.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+use("UnitTest");
+
+use("Demo")
```
Comment (thpdbc) [inline javascript/src/main/resources/javascript/Demo.test.js]: Kan Demo.*.js fjernes igen nu?

**Review by thpdbc — APPROVED** [review]
Overvej evt noget log.debug(...) med hvilke værdier der trækkes ud af metoder og hvilke kode branches den kommer ind i undervejs

**Review by thpdbc — APPROVED** [review]

**Review by mb-dbc-dk — APPROVED** [review]

File: javascript/README.javascript.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  "description" : null,
+  "pid" : "example",
+  "creators" : [ ],
+  "subjects" : [ ],
+  "types" : [ "Bog" ]
+}
+
+The pid corresponds to the folder name for the acceptance test. 
+
+
```
Comment (mb-dbc-dk) [inline javascript/README.javascript.md:107]: If a mismatch between expected and actual json exists, the actual will be located as a file under: target/test-classes/accept-test/{test-name}/actual.json, ready to copy if the semantics has changed, and the actual is correct.

File: javascript/README.javascript.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+You need to have installed the dbc-jsshell and postgresql
+
+
+## update environment
+When you've checked this project out from git the first thing you should do is go to the root and run 'mvn install'
+	> user@machine:~/work-presentation$ mvn install
+
+In the following development you should run this command each time you switch branches or the branch has been updated:
+
+	> user@machine:~/work-presentation$ mvn clean verify -DskipITs
```
Comment (mb-dbc-dk) [inline javascript/README.javascript.md]: clean install - for en sikkerheds skyld.

**Pull request merged**